pkg/app/utils.go

```diff
@@ -94,3 +96,21 @@ func bastionDebugWorkloadSpec(ctx context.Context, name string, image string, na
 		},
 	}
 }
+
+// getVpcIdForRDSInstance gets the VPC_ID from env if set or from the default VPC
+func getVpcIdForRDSInstance(ctx context.Context, ec2Cli *ec2.EC2) (string, error) {
+	vpcID := os.Getenv("VPC_ID")
+
+	// VPCId is not provided, use Default VPC
+	if vpcID == "" {
+		defaultVpc, err := ec2Cli.DescribeDefaultVpc(ctx)
+		if err != nil {
+			return "", err
+		}
+		if len(defaultVpc.Vpcs) == 0 {
+			return "", fmt.Errorf("No default VPC found")
+		}
+		return *defaultVpc.Vpcs[0].VpcId, nil
+	}
+	return vpcID, nil
```

Comment:
```suggestion
	// VPCId is not provided, use Default VPC
	if vpcID != "" {
	        return vpcID, nil
	}
	defaultVpc, err := ec2Cli.DescribeDefaultVpc(ctx)
	if err != nil {
		return "", err
	}
	if len(defaultVpc.Vpcs) == 0 {
		return "", fmt.Errorf("No default VPC found")
	}
	return *defaultVpc.Vpcs[0].VpcId, nil
```
[nit] for better readability